src/arch/xtensa/include/xtensa/core-macros.h

```diff
@@ -433,7 +433,7 @@ static inline unsigned  XTHAL_COMPARE_AND_SET( int *addr, int testval, int setva
 	    : "=a"(result) : "0" (setval), "a" (testval), "a" (addr)
 	    : "memory");
 #elif XCHAL_HAVE_INTERRUPTS
-    int tmp;
+    int tmp=0;
```

Comment:
This fix is not ACP70-specific at all so it must be in a separate, standalone commit. The error message should be in the commit message.